### DISCUSSION TITLE
Unable to reinstall

### DISCUSSION THREAD

**Discussion by jeremytregunna** [community/community#12212] [Copilot Conversations] [answered]
I was using an older version of Copilot and it told me to update, so I tried. Nothing happened after repeated attempts. So I installed copilot, and tried to reinstall but have not been able to find the extension anymore, the installation instructions to `ext install GitHub.copliot` no longer finds the extension.

Not sure what my options are here to fix the problem. I am authenticated with the correct GitHub account.

**Comment by davecheney** [comment, marked as answer]
https://marketplace.visualstudio.com/items?itemName=GitHub.copilot

**Reply by jeremytregunna** [reply]
Right, those are the installation instructions that don't work. When I paste that into the Ctrl+P field, it switches to the extensions view, shows "No extensions found." doesn't show me the extension like it did when I installed it the first time.

**Reply by davecheney** [reply]
What IDE are you using?

**Reply by jeremytregunna** [reply]
VSCode 1.64.2 on Linux

**Reply by davecheney** [reply]
Can you please double check that you are using VSCode, not OSS Code or VSCodium (those are not supported)

**Reply by jeremytregunna** [reply]
Did it ever work with the OSS build? It appears as though I had OSS Code installed for a long time. I can confirm I was able to reinstall it with the vscode bin. Can close.

**Reply by davecheney** [reply]
We've never supported any variant except VSCode. The internals of VSCode change often so it's a fast moving target. Sadly the other variant may work from time to time, but its not something we can support at the moment.
